### PR TITLE
drop type specs in RecoVertex RecoPixelVertexing

### PR DESCRIPTION
**Pull request by jeongeun**
#### PR description: 
Update the safer syntax for existing parameter :
- drop type specifications where the original parameter exists.
- move all parameter inside the clone

Instead of modifying parameters with full type specs, which can be interpreted as an insertion of a new parameter, it is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.
(The previous PR were [PR#31162](https://github.com/cms-sw/cmssw/pull/31162),[PR#31243](https://github.com/cms-sw/cmssw/pull/31243),[PR#31332](https://github.com/cms-sw/cmssw/pull/31332), [PR#31389](https://github.com/cms-sw/cmssw/pull/31389), [PR#31523](https://github.com/cms-sw/cmssw/pull/31523), [PR#31538](https://github.com/cms-sw/cmssw/pull/31538), [PR#31748](https://github.com/cms-sw/cmssw/pull/31748))

In this PR, total 5 files changed. 

- RecoVertex/Configuration  : 2 files 

- RecoPixelVertexing/PixelLowPtUtilities : 2 files
- RecoPixelVertexing/PixelTriplets :  1 file 

#### PR validation:
Event Content comparison check was also done and there is no change with these updates.
Tested in CMSSW_11_2_X, the basic test all passed in the [CMSSW PR instructions](https://cms-sw.github.io/PRWorkflow.html).

### CODE REVIEW

**Comment by perrotta** [comment]
please test

File: RecoPixelVertexing/PixelLowPtUtilities/python/ClusterShapeExtractor_cfi.py [hunk 1]
```diff
@@ -20,5 +20,5 @@
 
 from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
 phase2_tracker.toModify(clusterShapeExtractor,
-    pixelSimLinkSrc = cms.InputTag('simSiPixelDigis', 'Pixel'),
+    pixelSimLinkSrc = 'simSiPixelDigis:Pixel'),
```
Comment (perrotta) [inline RecoPixelVertexing/PixelLowPtUtilities/python/ClusterShapeExtractor_cfi.py:23]: indeed... A right parenthesis was left after the InputTag

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
+1
- Type specifications dropped as intended
- Jenkins tests pass and show no differences

**Comment by silviodonato** [comment]
+1

**Pull request merged**